.github/ISSUE_TEMPLATE/ask-question.yml

```diff
@@ -0,0 +1,50 @@
+name: 🙋‍♂️ Ask a question
+description: Tell us what's on your mind
+title: "[question]: "
+labels: ["triage"]
+# assignees:
+#   - assign your sdk team e.g., onesignal/unity
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Having issues integrating this SDK?
+  - type: textarea
+    id: question
+    attributes:
+      label: How can we help?
+      description: Specific question regarding integrating this SDK.
+      placeholder: How do I...?
+    validations:
+      required: true
+  - type: textarea
+    id: expo-sdk-version
+    attributes:
+      label: OneSignal Expo SDK version
+      description: The version of the OneSignal Expo SDK you have a question for.
+      placeholder: 1.0.0
+    validations:
+      required: true
+  - type: dropdown
+    id: platform
+    attributes:
+      label: Platform
+      description: Which platform are you having an issue with?
+      multiple: true
+      options:
+        - iOS
+        - Android
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: Shell
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](#insert-project-code-of-conduct-here)
```

Comment:
This should link to the SDK project code of conduct. I don't know where the code of conduct is or if we even have one. I created this issue three weeks ago; perhaps you can be the person on your team to make the decision: I'll support whatever y'all decide to do.